File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Char/System.Char/Char.IsPunctuation.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_95

	''' <summary>
	'''     Indicates whether the specified Unicode character is categorized as a punctuation mark.
	''' </summary>
	''' <param name="c">The Unicode character to evaluate.</param>
	''' <returns>true if  is a punctuation mark; otherwise, false.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function IsPunctuation(c As [Char]) As [Boolean]
		Return [Char].IsPunctuation(c)
	End Function
End Module

File: src/Z.Core/System.Char/System.Char/Char.IsPunctuation.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Indicates whether the specified Unicode character is categorized as a punctuation mark.
    /// </summary>
    /// <param name="c">The Unicode character to evaluate.</param>
    /// <returns>true if  is a punctuation mark; otherwise, false.</returns>
    public static Boolean IsPunctuation(this Char c)
    {
        return Char.IsPunctuation(c);
    }
}